spring-grpc-core/pom.xml

```diff
@@ -67,13 +67,19 @@
 			<artifactId>grpc-stub</artifactId>
 		</dependency>
 
+		<dependency>
```

Comment:
This should be optional? Or maybe it's already present transitively from Spring (so we don't need to declare it at all)?

Reply:
io.micrometer.core.instrument.binder.grpc.ObservationGrpcServerInterceptor would be needed for the AutoConfiguration and that's not included I guess as I found this is breaking if I take it out.